services/src/main/java/com/chatbot/services/IDMapping.java

```diff
@@ -0,0 +1,104 @@
+package com.chatbot.services;
```

Comment:
1. Subdirectory
2. Rename class as HangoutsIdMapping

Reply:
This mapping is not hangouts specific, the getters take chatClient as an argument.